src/index.html

```diff
@@ -1,30 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Meal db</title>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
-</head>
-<body>
+<
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+  </head>
+  <body>
     <div class="header">
-        <div class="logo">Meal Cards</div>
-        <ul class="links">
-            <li><a href="" class="meal-counter"></a> profile</li>
-            <li><a href="" class="meal-counter"></a> contact</li>
-        </ul>
+      <div class="logo">Meal Cards</div>
+      <ul class="links">
+        <li><a href="" class="meal-counter"></a></li>
+        <!-- <li><a href=""></a> contact</li> -->
+      </ul>
     </div>
-    <div class="grid-container">
-
-    </div>
-    <footer>
-        <div>
-            <p>
-                created by
-                <a href="http://github.com/lawrahkonwea" class="github-link">lawrahkonwea</a>
-                for Microverse Capstone Project
-            </p>
+    <div class="main"></div>
+    <div class="grid-container"></div>
+    <section class="popup-container">
+      <div class="popup-window">
+        <div class="close-btn">
+          <i id="close" class="fa-solid fa-xmark"></i>
+        </div>
+        <div class="popup-image">
+          <img src="." alt="popup-image" />
+    
         </div>
+        <div class="item-info">
+          <h3 class="dish-name"></h3>
+          <p class="instruction"></p>
+        </div>
+        <div class="comments-container">
+          <h4 class="comments-counter"></h4>
```

Comment:
- [ ] Great job in creating this project to this point. I noticed that when I add a comment to an item in the comment popup, the comment counter on the UI is not updated as expected. I think this is a core requirement for this project and should be correctly implemented. 
- [ ] The comment counter function should look for a specific DOM element which is `<ul class="comments-generator"></ul>` and make the counting based on what is actually displayed on the page. That means do something like this 👇 

```
let commentCount = document.querySelector(".comments-generator").children.length;
```

This will give the actually comments displayed on the screen. This is a hint. you will have to write out the function. That function will be called after the comment has been successfully added to the involvement API. In other words, it should be called after the comment submit button is clicked and the post request is successful. 

![image](https://user-images.githubusercontent.com/84156714/212485904-589dcb7d-5056-43af-8abb-61f94a546fee.png)